example/full-config.php

```diff
@@ -193,6 +196,20 @@
 
             DependencyFactory::class => DependencyFactoryFactory::class,
             ConfigurationLoader::class => ConfigurationLoaderFactory::class,
+
+            SchemaAssetsFilter::class => static function (): callable {
```

Comment:
`SchemaAssetsFilter` is not a real class here, is it? Let's use a string, if it isn't.